R/pkgdown_print.R

```diff
@@ -0,0 +1,16 @@
+#' Print object in pkgdown output
+#'
+#' @param x Object to display
+#' @param visible Whether it is visible or not
+#'
+#' @return character vector showing the value of x
+#' @export
+pkgdown_print <- function(x, visible) {
+  UseMethod("pkgdown_print")
+}
+
+#' @export
+pkgdown_print.default <- function(x, visible) {
+  if (visible)
+    if (isS4(x) && requireNamespace("methods")) methods::show(x) else print(x)
```

Comment:
I don't think you need to check that methods is installed here; if methods isn't installed, I think the user will have bigger problems 😄

Reply:
Also done.